distribution/lib/Standard/Base/0.0.0-dev/src/Data/Range.enso

```diff
@@ -53,18 +55,21 @@ type Range
         _ ->
             Error.throw (Illegal_Argument.Error "Range step should be an integer.")
 
+    ## Returns the first element that is included within the range or `Nothing`
+       if the range is empty.
+    first : Integer | Nothing
+    first self = if self.is_empty then Nothing else self.start
+
+    ## Returns the second element that is included within the range or `Nothing`
+       if the range has less than 2 element
+    second : Integer | Nothing
+    second self = if self.length < 2 then Nothing else self.start + self.step
```

Comment:
Why are we returning `Nothing` instead of `Index_Out_Of_Bounds` like in `Vector`/`List`?